README.md

```diff
@@ -17,7 +17,7 @@ What are constructor functions and the `new` keyword? What is a prototype? Descr
 
 Your Answer:
 ```text
-
+Constructor functions are used to create new objects in JavaScript.  The 'new' keyword is used when creating a new instance of the constructor object.  Prototypes are properties that exist on all functions and define what properties and methods are available to the object.  We could use a constructor to create several flower objects, perhaps with key-values such as color, region and species.  In addition to that, we might modify the constructor prototype to include properties the flower objects have in common, such as plant food given, hours of sunshine per day, etc.
```

Comment:
:+1: Looks good.  
I'm being a bit pedantic, but the Prototype is an object.  We add both properties and methods, most often methods, to the prototype.